### PR TITLE
fix punctuation mistyped

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

mistyped punctuation was found when reading documents, solved it by the way. 🧐

### CODE REVIEW

**Review by pigletfly — APPROVED** [review]
/lgtm

**Comment by tengqm** [comment]
Thank you for proposing this fix. However, we will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this **on the main branch**:

```shell
./scripts/lsync.sh content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md
```

谢谢贡献。根据[指南](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87)约定，我们会拒绝对页面的**部分**修改。部分修改针对的是对应页面的英文上游还有其他改动需要同步的情况。
一旦我们合并了此 PR，将来就无法根据文件的最后更改时间来判断是否有其他上游变更尚未同步。请理解。

请在 main 分支上执行以下命令查看上游尚未同步的更改：

```shell
./scripts/lsync.sh content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md
```


/hold

**Comment by kerthcet** [comment]
hey @tengqm , I checked the document comparing with En edition, it's all in sync. And  command `./scripts/lsync.sh content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md` confirmed it. What's more, I also updated some words to be more native, correct me if not. Thanks.

**Comment by kerthcet** [comment]
as `Dynamic Kubelet Configuration` is deprecated in release v1.22, I'll help to update the document later.

**Comment by kerthcet** [comment]
shoud I open another pr to update the document? then I'll close this pr. @tengqm

**Comment by kerthcet** [comment]
update to latest version.

**Comment by kerthcet** [comment]
/retitle keep in sync with deprecation of Dynamic Kubelet Configuration

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 1]
```diff
@@ -34,10 +42,10 @@ fields is available in the inline `KubeletConfiguration`
 [type documentation](https://github.com/kubernetes/kubernetes/blob/release-1.11/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go).
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:42]: 这行上游英文已经变了，没有同步。

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 2]
```diff
@@ -34,10 +42,10 @@ fields is available in the inline `KubeletConfiguration`
...
 -->
 {{< warning >}}
-所有 kubelet 配置参数都可以动态更改，但这对某些参数来说是不安全的。
-在决定动态更改参数之前，你需要深刻理解这种变化将如何影响你的集群的行为。
-在把一组变更推广到集群范围之前，需要在较小规模的节点集合上仔细地测试这些配置变化。
-与特定字段配置相关的建议可以在源码中 `KubeletConfiguration`
+所有 kubelet 配置参数都可以被动态更改，但对某些参数来说是不安全的。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:45]:
```suggestion
所有 kubelet 配置参数都可以被动态更改，但对某些参数来说这类更改是不安全的。
```

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 3]
```diff
@@ -34,10 +42,10 @@ fields is available in the inline `KubeletConfiguration`
...
+在决定动态更改参数之前，你需要深刻理解这个改动将会如何影响集群的行为。
+在将变更扩散到整个集群之前，你需要先在小规模的节点集合上仔细地测试这些配置变动。
+特定字段相关的配置建议可以在源码 `KubeletConfiguration`
 [类型文档](https://github.com/kubernetes/kubernetes/blob/release-1.11/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go)中找到。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:49]: 同上。

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 4]
```diff
@@ -95,11 +103,11 @@ Kubelet's configuration.
 2. Wrap this file in a ConfigMap and save it to the Kubernetes control plane.
 3. Update the Kubelet's corresponding Node object to use this ConfigMap.
 -->
-在运行中的集群中配置 kubelet 的基本工作流程如下：
+在运行的集群中配置 kubelet 的基本工作流程如下：
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:106]: 这句可以保持原文。

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 5]
```diff
@@ -95,11 +103,11 @@ Kubelet's configuration.
...
 
-1. 编写一个 YAML 或 JSON 的配置文件包含 kubelet 的配置。
+1. 编写一个包含 kubelet 配置的 YAML 或 JSON 文件。
 2. 将此文件包装在 ConfigMap 中并将其保存到 Kubernetes 控制平面。
-3. 更新 kubelet 的相应节点对象以使用此 ConfigMap。
+3. 更新 kubelet 所在节点对象使用此 ConfigMap。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:110]:
```suggestion
3. 更新 kubelet 所在节点对象以使用此 ConfigMap。
```

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 6]
```diff
@@ -121,7 +129,7 @@ and is overridden by command-line flags. Unspecified values in the new configura
 will receive default values appropriate to the configuration version
 (e.g. `kubelet.config.k8s.io/v1beta1`), unless overridden by flags.
 -->
-这个新配置完全地覆盖 `--config` 所提供的配置，并被命令行标志覆盖。
+新配置将会完全地覆盖 `--config` 所提供的配置，并被命令行标记覆盖。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:132]: flag 统一译为标志，标记的英文是 tag

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 7]
```diff
@@ -158,8 +166,8 @@ as immutable by convention, aided by `kubectl`'s `-append-hash` option,
 and incrementally roll out updates to `Node.Spec.ConfigSource`.
 -->
 {{< warning >}}
-通过就地更新 ConfigMap 来更改配置是 *可能的*。
-尽管如此，这样做会导致所有配置为使用该 ConfigMap 的 kubelet 被同时更新。
+尽管通过原地更新 ConfigMap 来更改配置是 *可能的*。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:169]: 就地更新更好些。

**Comment by tengqm** [comment]
还有如下变更没有同步：

```
@@ -356,7 +361,7 @@ metadata and checkpoints. The structure of the kubelet's checkpointing directory
 
-## Understanding Node.Status.Config.Error messages {#understanding-node-config-status-errors}
+## Understanding `Node.Status.Config.Error` messages {#understanding-node-config-status-errors}
 
 The following table describes error messages that can occur
 when using Dynamic Kubelet Config. You can search for the identical text
@@ -378,6 +383,10 @@ internal failure, see Kubelet log for details | The kubelet encountered some int
 
 ## {{% heading "whatsnext" %}}
 
- - For more information on configuring the kubelet via a configuration file, see
+- For more information on configuring the kubelet via a configuration file, see
 [Set kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file).
 - See the reference documentation for [`NodeConfigSource`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#nodeconfigsource-v1-core)
+- Learn more about kubelet configuration by checking the
+  [`KubeletConfiguration`](/docs/reference/config-api/kubelet-config.v1beta1/)
+  reference.
+
```

**Comment by kerthcet** [comment]
hey @tengqm , finished all the tips, thanks for your patience.

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 8]
```diff
@@ -13,32 +13,40 @@ content_type: task
...
 <!--
 [Dynamic Kubelet Configuration](https://github.com/kubernetes/enhancements/issues/281)
 allows you to change the configuration of each Kubelet in a live Kubernetes
 cluster by deploying a ConfigMap and configuring each Node to use it.
 -->
 [动态 kubelet 配置](https://github.com/kubernetes/enhancements/issues/281)
-允许你在一个运行的 Kubernetes 集群上通过部署 ConfigMap
-并配置每个节点来使用它来更改每个 kubelet 的配置，。
+允许你通过部署一个所有节点都会使用的 ConfigMap 
+达到在运行的 Kubernetes 集群中更改 kubelet 配置的目的。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:33]:
```suggestion
达到在运行中的 Kubernetes 集群中更改 kubelet 配置的目的。
```

**Comment by tengqm** [comment]
貌似如下变更仍没有同步：

@@ -356,7 +361,7 @@ metadata and checkpoints. The structure of the kubelet's checkpointing directory
 
-## Understanding Node.Status.Config.Error messages {#understanding-node-config-status-errors}
+## Understanding `Node.Status.Config.Error` messages {#understanding-node-config-status-errors}
 
 The following table describes error messages that can occur
 when using Dynamic Kubelet Config. You can search for the identical text
@@ -378,6 +383,10 @@ internal failure, see Kubelet log for details | The kubelet encountered some int
 
 ## {{% heading "whatsnext" %}}
 
- - For more information on configuring the kubelet via a configuration file, see
+- For more information on configuring the kubelet via a configuration file, see
 [Set kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file).
 - See the reference documentation for [`NodeConfigSource`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#nodeconfigsource-v1-core)
+- Learn more about kubelet configuration by checking the
+  [`KubeletConfiguration`](/docs/reference/config-api/kubelet-config.v1beta1/)
+  reference.
+

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md (hunk 8 above)
Comment (kerthcet) [reply content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md]: done. :)

**Comment by kerthcet** [comment]
> 貌似如下变更仍没有同步：
> 
> @@ -356,7 +361,7 @@ metadata and checkpoints. The structure of the kubelet's checkpointing directory
> 
> -## Understanding Node.Status.Config.Error messages {#understanding-node-config-status-errors}
> +## Understanding `Node.Status.Config.Error` messages {#understanding-node-config-status-errors}
> 
> The following table describes error messages that can occur
> when using Dynamic Kubelet Config. You can search for the identical text
> @@ -378,6 +383,10 @@ internal failure, see Kubelet log for details | The kubelet encountered some int
> 
> ## {{% heading "whatsnext" %}}
> * * For more information on configuring the kubelet via a configuration file, see
>     +- For more information on configuring the kubelet via a configuration file, see
>     [Set kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file).
> * See the reference documentation for [`NodeConfigSource`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#nodeconfigsource-v1-core)
>   +- Learn more about kubelet configuration by checking the
> 
> * [`KubeletConfiguration`](/docs/reference/config-api/kubelet-config.v1beta1/)
> * reference.

double checked again. I suppose it's finished.

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 9]
```diff
@@ -646,11 +654,15 @@ internal failure, see Kubelet log for details | 在对配置进行同步的循
...
 - See the reference documentation for [`NodeConfigSource`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#nodeconfigsource-v1-core)
+- Learn more about kubelet configuration by checking the
+  [`KubeletConfiguration`](/docs/reference/config-api/kubelet-config.v1beta1/)
+  reference.
 -->
 - 关于如何通过配置文件来配置 kubelet 的更多细节信息，可参阅
   [使用配置文件设置 kubelet 参数](/zh/docs/tasks/administer-cluster/kubelet-config-file).
 - 阅读 API 文档中 [`NodeConfigSource`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#nodeconfigsource-v1-core) 说明
-
+- 查阅[`KubeletConfiguration`](/docs/reference/config-api/kubelet-config.v1beta1/)文献了解更多关于 kubelet
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:667]: learn more about sth. --> 进一步了解某物，不要翻译成“了解更多关于”

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 10]
```diff
@@ -609,7 +617,7 @@ metadata and checkpoints. The structure of the Kubelet's checkpointing directory
 ```
 
 <!-- 
-## Understanding Node.Status.Config.Error messages {#understanding-node-config-status-errors}
+## Understanding `Node.Status.Config.Error` messages {#understanding-node-config-status-errors}
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:620]: 626 行应相应改为

```
## 理解 `Node.Status.Config.Error` 消息  {#understanding-node-config-status-errors}
```

**Review by tengqm — COMMENTED** [review]
LGTM. Two nits.

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md (hunk 9 above)
Comment (kerthcet) [reply content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md]: thanks for correct me.

File: content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md (hunk 10 above)
Comment (kerthcet) [reply content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md:620]: done :)

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**